modules/140-user-authz/crds/clusterauthorizationrule.yaml

```diff
@@ -150,5 +153,199 @@ spec:
+
+                      **Caution!** since `ClusterAdmin` can edit `ClusterRoleBindings`, he can broader his privileges within the cluster;
+                    * `SuperAdmin` — can perform any actions with any objects (note that `limitNamespaces` and `namespaceSelector` (see below) restrictions remain valid).
+                  enum: [User,PrivilegedUser,Editor,Admin,ClusterEditor,ClusterAdmin,SuperAdmin]
+                  example: 'PrivilegedUser'
+                portForwarding:
+                  type: boolean
+                  default: false
+                  description: |
+                    Allow/disallow the user to do `port-forwarding`.
+                allowScale:
+                  type: boolean
+                  default: false
+                  description: |
+                    Defines if scaling of Deployments and StatefulSets is allowed/not allowed.
+                allowAccessToSystemNamespaces:
+                  type: boolean
+                  description: |
+                    Allow access to System namespaces (kube-*, d8-*, loghouse, default).
+
+                    Option available **only** if the [enableMultiTenancy](configuration.html#parameters-enablemultitenancy) option is enabled.
+                  x-doc-d8Revision: ee
+                  x-doc-default: false
+                namespaceSelector:
+                  description: |
+                    Defines a set of namespaces accessible by the user, using labelSelector values.
+
+                    Compatibility with limit limitNamespaces:
+                    * If this field is set, the limitNamespaces value is ignored for the current clusterAuthorizationRule.
+                    * The field limitNamespaces is deprecated.
+
+                    The decision making process:
+                    * If the field is defined, then only the namespaces matching labelSelector's criteria are accessible (except for the system ones - see `spec.allowAccessToSystemNamespaces` below).
+                    * If the field is not defined, then all namespaces are accessible (except for the system ones - see `spec.allowAccessToSystemNamespaces` below).
+
+                    Option available only if [enableMultiTenancy](configuration.html#parameters-enablemultitenancy) option is enabled.
+                  type: object
+                  required:
+                    - labelSelector
+                  properties:
+                    labelSelector:
+                      description: |
+                         Contains labelselector definitions.
+
+                         If both matchExpressions and matchLabels parameters are set, their requirements are ANDed together — they must all be satisfied in order to match.
+                      type: object
+                      anyOf:
+                        - required: [ matchLabels ]
+                        - required: [ matchExpressions ]
```

Comment:
If we use it somewhere in Deckhouse, this is a mistake. Kubernetes doc says that this is ok to specify both.
https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements

We can leave it out of the scope of this PR but fix it in the future everywhere in Dechouse.